### PR TITLE
Add note about view-source requests

### PR DESCRIPTION
**Pull request by caitmuenster**
Added note explaining `view-source:` requests may be matched based on the inner URL.

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/webrequest/requestfilter/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2 id="Type">Type</h2>
 
 <dl class="reference-values">
  <dt><code>urls</code></dt>
- <dd><code>array</code> of <code><code>string</code></code>. An array of <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns">match patterns</a>. The listener will only be called for requests whose targets match any of the given patterns. Only requests made using HTTP or HTTPS will trigger events, other protocols (such as data: and file:) supported by pattern matching do not trigger events.</dd>
+ <dd><code>array</code> of <code><code>string</code></code>. An array of <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns">match patterns</a>. The listener will only be called for requests whose targets match any of the given patterns. Only requests made using HTTP or HTTPS will trigger events, other protocols (such as data: and file:) supported by pattern matching do not trigger events. `view-source:` requests may be matched based on the inner URL.</dd>
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/webrequest/requestfilter/index.html:25]: the inner URL -> its inner URL.

**Review by Rob--W — APPROVED** [review]

File: files/en-us/mozilla/add-ons/webextensions/api/webrequest/requestfilter/index.html (hunk 1 above)
Comment (caitmuenster) [reply files/en-us/mozilla/add-ons/webextensions/api/webrequest/requestfilter/index.html]: Thanks @Rob--W!

**Review by Rob--W — APPROVED** [review]

**Comment by caitmuenster** [comment]
Looks like this needs a second review to merge. @zombie, would you mind giving this a quick r?

**Review by zombie — APPROVED** [review]
lgtm

**Comment by zombie** [comment]
Err, I'm not sure these are post-processed anymore, might need `<code>view-source</code>` tags instead of ` backticks.

**Comment by chrisdavidmills** [comment]
> Err, I'm not sure these are post-processed anymore, might need `<code>view-source</code>` tags instead of ` backticks.

Correct — the page content is not processed at all, except for the meta content in the header and the KumaScript macro calls.

**Comment by caitmuenster** [comment]
Thanks all!

**Pull request merged**